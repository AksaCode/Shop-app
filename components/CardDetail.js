import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

const CardDetail = ({ title, price }) => {
  return (
    <View style={styles.content}>
      <Text style={styles.title}>{title}</Text>
      <Text style={styles.price}>${price}</Text>
    </View>
  );
};

export default CardDetail;

const styles = StyleSheet.create({
  content: {
    alignItems: 'center',
    justifyContent: 'center',
  },
  title: {
    fontSize: 20,
    fontFamily: 'open-sans-bold',
  },
  price: {
    fontSize: 15,
    fontFamily: 'open-sans',
  },
});
