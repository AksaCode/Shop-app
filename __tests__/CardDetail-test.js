import React from 'react';
import renderer from 'react-test-renderer';
import CardDetail from '../components/CardDetail';

test('renders correctly', () => {
  const tree = renderer.create(<CardDetail />).toJSON();
  expect(tree).toMatchSnapshot();
});
