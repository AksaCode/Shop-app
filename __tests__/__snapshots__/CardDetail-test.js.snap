// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "open-sans-bold",
        "fontSize": 20,
      }
    }
  />
  <Text
    style={
      Object {
        "fontFamily": "open-sans",
        "fontSize": 15,
      }
    }
  >
    $
  </Text>
</View>
`;
